### ISSUE TITLE
Sidebar problem

### ISSUE THREAD

**Issue by 8513421** [hughbris/grav-theme-landed#9]
Hi! Sorry but i dont understand your previous explanation. Could you write me exact code to put in home yaml section to run sidebar in home page? Becasue in demo pages home-sidebars is inactivated (and no code for this) and i spend few hours to figured out how to activated it with no result. Thanks in advance!

_Originally posted by @8513421 in https://github.com/hughbris/grav-theme-landed/issues/7#issuecomment-615286180_

**Comment by hughbris** [comment]
Sorry this is nearly four years later and that you wasted a few hours trying to figure this out. (Four years ago. Damn!) The explanation is confusing and yes this needs to be better documented.

It probably no longer matters to you, but I will wait until I have completed #2 before documenting my setup. One reason is that I sometimes require small (sometimes breaking) changes to frontmatter structure when creating blueprints. Another reason is that the model will be clearer in my head after making blueprints and the explanation will be better.

**Comment by ganar** [comment]
Sorry to bother you with this after so much time: I'm having the same issue with a clean Grav CMS  installation and the template. Can't make the sidebars work at home.

I added the code in the YAML —`system.yaml` and the top of the pages— and copied the _demo content to the root of the content folder, but still does not work for me.

**Comment by hughbris** [comment]
@ganar: So from your comment in #7, did you sort this out too? I didn't realise this had been split into two issues for some reason. I think you didn't realise originally either.

**Comment by ganar** [comment]
Yes I did: the problem is that I was adding the code to the wrong configuration file: instead of the `system.yaml` it had to be done in the `site.yaml`, as instructed. 

My bad!
